docs/src/release-notes-python.md

```diff
@@ -21,9 +21,6 @@ title: "Release notes"
 - [Intro](./intro.md)
 - [Authentication](./auth.md)
 - [Chome Extensions](./chrome-extensions.md)
-- [Playwright Test Annotations](./test-annotations.md)
```

Comment:
All the updated docs in the release-notes which got removed are language specific.